browser/src/control/jsdialog/Widget.OverflowGroup.ts

```diff
@@ -44,6 +45,39 @@ function setupOverflowMenu(
 		'.ui-overflow-group-label',
 	) as HTMLElement;
 
+	let expanderIconRightDiv: HTMLElement | undefined;
+
+	if (more) {
+		const moreOptionsButton = document.createElement('button');
+		moreOptionsButton.className = 'ui-content unobutton';
+		moreOptionsButton.id = `More-${id}`;
```

Comment:
Can we use similar prefix as all other unfolded nodes around:    `${id}-more-button`
So single "search" in DOM will find all related nodes.

Also in the class list something we can address: `ui-overflow-group-more`